### PR TITLE
Added EVE University, Abiotic Factor wiki.gg, & Extra Doctor Who Fandom Redirects

### PR DESCRIPTION
**Pull request by DrWhoFan13**
Added:

- A redirect from the EVE Online Fandom to EVE University Wiki, a wiki run by a faction of EVE Online players dedicated to educating new players.

- A redirect from Torchwood's Archived Fandom to Tardis Wiki, as that has superior coverage of the Torchwood subfranchise.

- Several more Doctor Who Fandoms to redirect to Tardis Wiki, because that again has superior coverage than the redirected wikis.

- A redirect to Abiotic Factor wiki.gg

### CODE REVIEW

File: data/sitesEN.json [hunk 1]
```diff
@@ -1701,14 +1701,20 @@
   },
   {
     "id": "en-doctorwho",
-    "origins_label": "Tardis Fandom & Doctor Who Neoseeker Wikis",
+    "origins_label": "Whoniverse Fandom & Neoseeker Wikis",
```
Comment (KevinPayravi) [inline data/sitesEN.json:1704]: What do you think of setting the label to "Doctor Who" instead of "Whoniverse"?

Comment (DrWhoFan13) [reply data/sitesEN.json:1704]: A bit inaccurate, as :
- "Whoniverse" is a term to refer to the entire Doctor Who franchise. 
-  TARDIS wiki covers all Doctor Who content produced by the BBC regardless of canonicity (as much as Doctor Who can have a canon) as demonstrated by this page for a [spoof Seventh Doctor](https://tardis.wiki/wiki/Seventh_Doctor_(The_Lenny_Henry_Show))

**Comment by DrWhoFan13** [comment]
Upon further thought, I changed it to "Doctor Who Fandom & Neoseeker Wikis"

File: data/sitesEN.json [hunk 2]
```diff
@@ -2220,6 +2266,25 @@
...
+        "origin": "EVE Fandom Wiki",
+        "origin_base_url": "eve.fandom.com",
+        "origin_content_path": "/wiki/",
+        "origin_main_page": "Main_Page"
+      }
+    ],
+    "destination": "EVE University Wiki",
+    "destination_base_url": "wiki.eveuniversity.org",
+    "destination_platform": "mediawiki",
+    "destination_icon": "eveonline.png",
```
Comment (KevinPayravi) [inline data/sitesEN.json:2283]: Missing favicon

**Review by KevinPayravi — APPROVED** [review]
Thank you!

**Pull request merged**